src/components/Filters/index.js

```diff
@@ -211,7 +211,7 @@ class Filters extends Component {
               </form>
               <button
                 className='search-btn'
-                onClick={ () => this.findSchools() }
+                onClick={ () => { this.findSchools(),  patchUser({ street_address: document.getElementById('homeAddress').value }, CurrentUser[0].id, { id: CurrentUser[0].id, username: CurrentUser[0].username, street_address: document.getElementById('homeAddress').value, email: CurrentUser[0].email, oath_id: CurrentUser[0].oath_id })}}
```

Comment:
this is ugly for now, just trying to get this working.  this being updating store with the users address they just ran a search with